src/main/python/hydra/lib/hydrabase.py

```diff
@@ -666,6 +671,66 @@ def get_mesos_slave_ips_attr(self, attr_type, attr_value):
         """
         return self.__mesos.get_slave_ips_from_attribute(attr_type, attr_value)
 
+    def get_all_mesos_slave_attr(self):
```

Comment:
not sure I understand the comment, what data structure to info list are we talking about here ?